### DISCUSSION TITLE
how to wire the client to a pty process?

### DISCUSSION THREAD

**Discussion by yunfan** [jquast/telnetlib3#105] [Ideas]
hi, thanks for the wonderful lib. can you give a example showing how to implmenet the traditional telnetd?

**Comment by jquast** [comment]
I will try

**Comment by jquast** [comment]
@yunfan I know this is a long time, but pty process with telnetlib3-server is now possible, see example in README, https://github.com/jquast/telnetlib3?tab=readme-ov-file#pty-execution

     telnetlib3-server --pty-exec /bin/bash -- --login

This is in "master" branch and will release to pypi within the next few days.

**Reply by yunfan** [reply]
thank you very much, but in this very long time, i had implment my own tmux like app, that means i already go the basic principle :D

any way i could still learn from your library
